config/values-schema.yml

```diff
@@ -0,0 +1,64 @@
+#@data/values-schema
+---
```

Comment:
hey @sethiyash , I guess maybe there was a misunderstanding here. But this is our public values file. So when I go to install KC via a package install I can configure all these values.

But they do nothing right now, so they actually have less value by including them.

Take SGC for example: when I choose an updateStrategy in my values file, the deployment is changed based on this:

https://github.com/carvel-dev/secretgen-controller/blob/develop/config/package-bundle/config/deployment.yml#L12

I think we want something similar here, either via overlays or via branching (i.e ifs/ switches whatever) directly in the deployment.yml